### DISCUSSION TITLE
async_connect for crdb?

### DISCUSSION THREAD

**Discussion by gordthompson** [psycopg/psycopg#638] [Q&A] [answered]
https://github.com/psycopg/psycopg/blob/1aac5322696480766b3c05421de0435da211e2bc/psycopg/psycopg/crdb/__init__.py#L7-L11

Should we have

```python
async_connect = AsyncCrdbConnection.connect 
```

in here as well?

**Comment by dvarrazzo** [comment, marked as answer]
Why, there is no `async_connect()` in the main module either. You can use `AsyncCrdbConnection.connect()`. The `connect()` alias for a sync connection is for DBAPI compliance only.
